packages/core/src/components/graphs/radar.ts

```diff
@@ -0,0 +1,463 @@
+// Internal Imports
+import { Component } from "../component";
+import { DOMUtils } from "../../services";
+import * as Configuration from "../../configuration";
+import { Events, TooltipTypes, Roles } from "../../interfaces";
+import { Tools } from "../../tools";
+import {
+	Point,
+	Angle,
+	radialLabelPlacement,
+	radToDeg,
+	polarToCartesianCoords,
+	distanceBetweenPointOnCircAndVerticalDiameter
+} from "../../services/angle-utils";
+
+// D3 Imports
+import { select } from "d3-selection";
+import { scaleBand, scaleLinear, ScaleLinear } from "d3-scale";
+import { max, extent } from "d3-array";
+import { lineRadial, curveLinearClosed } from "d3-shape";
+
+// used to make transitions
+let oldYScale: ScaleLinear<number, number>;
+
+export class Radar extends Component {
+	type = "radar";
+	svg: SVGElement;
+	groupMapsTo: string;
+	uniqueKeys: string[];
+	uniqueGroups: string[];
+	displayDataNormalized: any;
+	groupedDataNormalized: any;
+
+	init() {
+		const { events } = this.services;
+		// Highlight correct line legend item hovers
+		events.addEventListener(Events.Legend.ITEM_HOVER, this.handleLegendOnHover);
+		// Un-highlight lines on legend item mouseouts
+		events.addEventListener(Events.Legend.ITEM_MOUSEOUT, this.handleLegendMouseOut);
+	}
+
+	render(animate = true) {
+		this.svg = this.getContainerSVG();
+		const { width, height } = DOMUtils.getSVGElementSize(this.parent, { useAttrs: true });
+
+		const data = this.model.getData();
+		const displayData = this.model.getDisplayData();
+		const groupedData = this.model.getGroupedData();
+		const options = this.model.getOptions();
+		const configuration = Configuration.options.radarChart.radar;
+		const groupMapsTo = options.data.groupMapsTo;
+		const { angle, value } = options.radar.axes;
```

Comment:
```suggestion
		const configuration = Tools.getProperty(Configuration, "options", "radarChart", "radar");
		const groupMapsTo = Tools.getProperty(options, "data", "groupMapsTo");
		const { angle, value } = Tools.getProperty(options, "radar", "axes");
```
most of the codebase uses the helper function for accesing properties especially when the property paths are longer